cdap-master/src/test/java/co/cask/cdap/master/startup/ConfigurationCheckTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.master.startup;
+
+import co.cask.cdap.common.conf.CConfiguration;
+import co.cask.cdap.common.conf.Constants;
+import co.cask.cdap.common.guice.ConfigModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Tests Configuration for Kafka topic
+ */
+public class ConfigurationCheckTest {
+  private static ConfigurationCheck configurationCheck;
+
+  @BeforeClass
+  public static void setContext() {
+    CConfiguration cConf = CConfiguration.create();
+    cConf.set(Constants.Logging.KAFKA_TOPIC, "invalid:topic");
```

Comment:
nit: there is a small problem in this test. It tests for either one of these configurations to be valid, but not both, right? To make it stricter, we should validate with different cConf objects, one that has an invalid Log Kafka topic, another that has an invalid Notifications Kafka Topic.